Make the 'complete system setup' work again

schmir-at-bb opened the pull request:
We needed to make various changes to make this work again:
- use TOML configuration file for relay server
- create key for use in the relay server
- refer to existing addresses in laika

schmir committed:
Make the 'complete system setup' work again

We needed to make various changes to make this work again:
- use TOML configuration file for relay server
- create key for use in the relay server
- refer to existing addresses in laika

weilbith commented on docker/trustlines/config.toml:
```diff
@@ -0,0 +1,35 @@
+[relay]
```
I don't care too much, but why did the configuration got that much extended in comparison the previous version? Are this defaults or they related to the new setup by the documentation to provide the user with a quick solution to go for?

schmir-at-bb replied:
Well, most of them are new options.

schmir committed:
Fix key generation

Generating the keys with docker-compose doesn't work anymore, since we
mount the keystore related files into the container.

So, use docker to generate the key and do this before trying to start
the relay server image.

schmir-at-bb commented on docker/trustlines/README.md:
````diff
@@ -1,58 +1,117 @@
+
+Please copy them to the current directory with:
 ```
 docker-compose run --rm --no-deps -v $(pwd):/here --entrypoint /bin/bash relay -c "cp /opt/relay/trustlines-contracts/build/contracts.json /here"
````
I've fixed this by calling docker instead of docker-compose and doing it before using the relay image via docker-compose.

weilbith replied:
It works :+1:

schmir-at-bb commented:
Please take another look at it.
